galsim/phase_psf.py

```diff
@@ -827,14 +827,35 @@ def _reset(self):
                 pass
         self._update_attrs()
 
-    def instantiate(self, _bar=None, **kwargs):
-        for layer in self:
-            try:
-                layer.instantiate(**kwargs)
-            except AttributeError:
-                pass
-            if _bar:  # pragma: no cover
-                _bar.update()
+    def instantiate(self, pool=None, _bar=None, **kwargs):
+        """Instantiate the screens in this PhaseScreenList.
+
+        @param pool      A multiprocessing.Pool object to use to instantiate screens in parallel.
+        @param **kwargs  Keyword arguments to forward to screen.instantiate().
+        """
+        if pool is not None:
+            import sys
```

Comment:
This import is pretty lightweight, so it probably belongs at the top of the file (as per PEP8).  I don't mind having specialized imports inside functions or if clauses, but I think it's better form to move the easy/standard-lib ones up to the top.